README.md

```diff
@@ -103,10 +112,18 @@ func main() {
     log.Fatal(err)
   }
   
-  err = server.Serve()
-  if err != nil {
-    log.Fatal(err)
-  }
+
+  go func() {
```

Comment:
The server.Serve() doesn't actually need to be Go routine, does it? I suppose it doesn't matter much either way, but for the sake of keeping the readme example simple?

Reply:
It technically does not; however, in many of code [examples](https://github.com/mochi-mqtt/server/tree/fix-readme-example/examples) it is done with a go routine so I did this to remain consistent.

Reply:
I have no strong opinion either way. Perhaps we could wait to hear from @mochi-co to see if there was an original intent behind using the new go routine? I could see a world in which that was left over from a time when `.Serve()` did block which made it more necessary for the new routine.